### PR TITLE
Replaced deprecated "Formatter::formatResponse()" with "formatResponseForRequest" method

### LINKED ISSUE #667: formatResponse() function deprecated since 1.13 (php-http/message)
## Description
`formatResponse()` of class `Http\Message\Formatter` is deprecated since php-http/message v1.13, use `formatResponseForRequest() `instead.
Need to replace `formatResponse()` to `formatResponseForRequest()` used in `apigee_edge_debug` module.

Reference : https://github.com/php-http/message/releases/tag/1.13.0

## Actual Behavior
CircleCi `run-d9-check` failing with error 
`Call to deprecated method formatResponse() of class Http\Message\Formatter: since 1.13, use formatResponseForRequest() instead`

## Expected Behavior
CircleCi `run-d9-check` successfully completed.

### PR DESCRIPTION
**Pull request by shishir-intelli**
Closes #667 
Replaced deprecated `Formatter::formatResponse()` with `formatResponseForRequest` method.
Reference : https://github.com/php-http/message/releases/tag/1.13.0

### CODE REVIEW

File: modules/apigee_edge_debug/src/Plugin/DebugMessageFormatter/DebugMessageFormatterPluginBase.php [hunk 1]
```diff
@@ -155,7 +155,7 @@ public function formatResponse(ResponseInterface $response, RequestInterface $re
         }
       }
     }
-    return $this->getFormatter()->formatResponse($response);
+    return $this->getFormatter()->formatResponseForRequest($response, $request);
```
Comment (phdhiren) [inline modules/apigee_edge_debug/src/Plugin/DebugMessageFormatter/DebugMessageFormatterPluginBase.php:158]: This was just introduced in php-http/message:1.13
First need to update in apigee/apigee-client-php with this dependency version and that version should be updated here as well.

**Review by phdhiren — CHANGES_REQUESTED** [review]

File: modules/apigee_edge_debug/src/Plugin/DebugMessageFormatter/DebugMessageFormatterPluginBase.php (hunk 1 above)
Comment (shishir-intelli) [reply modules/apigee_edge_debug/src/Plugin/DebugMessageFormatter/DebugMessageFormatterPluginBase.php:158]: PR https://github.com/apigee/apigee-client-php/pull/207

Comment (shishir-intelli) [reply modules/apigee_edge_debug/src/Plugin/DebugMessageFormatter/DebugMessageFormatterPluginBase.php:158]: PR (apigee_edge-drupal) https://github.com/apigee/apigee-edge-drupal/pull/675

**Review by phdhiren — APPROVED** [review]
LGTM

**Pull request merged**